src/budy/static/css/report.css

```
@@ -0,0 +1,569 @@
+    font-family: "IBM Plex Mono", monospace;
+    font-size: 11px;
+    font-weight: 500;
+    letter-spacing: 0.1em;
+    margin-bottom: 16px;
+    text-transform: uppercase;
+}
+
+/* -- Definition list -- */
+.report-dl {
+    display: grid;
+    grid-column-gap: 16px;
```

Stylelint (17.9.0)

[error] Expected "grid-column-gap" to be "column-gap" (property-no-deprecated)

(property-no-deprecated)